### DISCUSSION TITLE
Only list certain entities, based on another entity state

### DISCUSSION THREAD

**Discussion by SkywaveLincoln** [jazzyisj/unavailable-entities-sensor#52] [Ideas]
Hi

I love this! It gives me a quick overview of problems I need to fix :)

One thing I would like to request (either to be implemented, or to have pointed out if I have missed it)... I have a Bambu printer with a ton of entities. I also have a smart plug to turn my printer on or off. If the printer is off, all the entities are shown. What would be nice, is if these entities could only be listed if the smart plug is on. 

Is it possible?

Thanks in advance.

**Comment by jazzyisj** [comment]
In cases like these where you expect entities to have an unknown state sometimes, I would either exclude those entities in the template or put them into the ignored entities group.   A common example of this is browser_mod entities that are all unknown when the device is offline.

You can create an entirely different template sensor to monitor your printer and it's entities if necessary.
Eg - Printter Error - If printer switch is on AND ( entity1 is unknown, OR entity 2 is unknown...) then true

**Reply by SkywaveLincoln** [reply]
I appreciate when people show me that there are solutions on the other side of the box. 

Thank you! This is a great suggestion!
